admin/sql/create_tables.sql

```diff
@@ -1,11 +1,12 @@
 BEGIN;
 
 CREATE TABLE lowlevel (
-  id          SERIAL,
-  mbid        UUID NOT NULL,
-  build_sha1  TEXT NOT NULL,
-  lossless    BOOLEAN                  DEFAULT 'n',
-  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+  id              SERIAL,
+  mbid            UUID NOT NULL,
```

Comment:
We can't replace `id`, as we have duplicate `mbid`s with multiple `id`s, and `id` is also used as an FK into the ll_json and hl tables.
MusicBrainz uses `gid` for this field (global ID?), which might make more sense, but may be confusing (it would mean one thing in AB and another thing in MB)